python/tidy/servo_tidy/tidy.py

```diff
@@ -723,13 +725,32 @@ def parse_config(content):
     config['ignore']['directories'] = map(lambda path: os.path.join(*path.split('/')),
                                           config['ignore']['directories'])
 
+    # Add dict of dir, list of expected ext to config
+    dirs_for_ext = config_file.get("check_dirs_for_exts", {})
```

Comment:
OK I have found a solution, I think, I'll push the code and we'll be able to discuss in more concrete way :)